src/components/MermaidDash/components/DownloadModal.jsx

```
@@ -1,16 +1,134 @@
+
+  const collectionMethods = Object.entries(DOWNLOAD_METHODS)
+
+  const _resetModalModeWhenModalOpenOrClose = useEffect(() => {
+    if (modalOpen) {
+      setErrorMessage('')
+      setModalMode(activeProjectCount === 0 ? 'no data' : 'download')
+      setSelectedMethod(getSurveyedMethod(surveyedMethodCount))
+    } else {
+      setModalMode('')
+    }
+  }, [modalOpen, activeProjectCount])
```

ESLint

[error] React Hook useEffect has a missing dependency: 'surveyedMethodCount'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)